azavea/cac-tripplanner
# Style tour map features

## Overview

- Style tour location remove button
- Style tour itinerary back button
- Add tour sidebar "spine"
- Style marker and route colors in tour, event, and explore map

Note marker color change in explore mode, to match colors for tour and event maps:
- Location: light gray (formerly orange in explore mode; low opacity retained for outside travelshed)
- Highlighted location: dark blue (formerly light blue in explore mode)
- Highlighted route leg: dark blue

### Demo

![image](https://user-images.githubusercontent.com/128699/67799575-fd4a8c80-fa5b-11e9-8ece-cfd1facd5584.png)

![image](https://user-images.githubusercontent.com/128699/67799520-e0ae5480-fa5b-11e9-8651-50e9f3cb7b7f.png)

### Notes

I considered a number of color options for markers and routes. We are limited to the colors offered by the Awesome Markers library. Green, Blue, and Red were the best options from a sheer visual design perspective (brightness, distinctness from other markers). 

Green wasn't viable because we already use if for the origin in directions and tour mode. 

Red makes semantic sense, since we use it for destination in directions mode, and in tour mode the highlighted color represents a tentative destination, but when I tried it, the red was simply too errory.

Blue works nicely because it's our primary brand color. The one problem is that we already use it to represent walking in directions mode. But since tour mode is distinct from directions mode, and given the constraints at hand, I think overloading the color is reasonable.

## Testing Instructions

 * Desktop
   * Go to a tour map
   * Note map marker and route colors
   * Note sidebar spine
   * Hover a sidebar card to see remove button, highlighted spine section, and map marker and route colors
   * Click Directions on a sidebar card to open Itinerary mode. Note back button.
   * Go to an event map. Note marker and route colors. Note absence of sidebar spine.
   * Go to explore mode. Note marker colors, with and without travelshed.
 * Mobile
   * Remove button, sidebar spine, and itinerary back button should all be hidden in mobile mode. 

## Checklist
